### PR TITLE
zh: TLS between TiDB components

### PR DESCRIPTION
**Pull request by weekface**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/enable-tls-between-components.md [hunk 1]
```diff
@@ -0,0 +1,1068 @@
...
+
+        然后部署 Drainer 集群：
+
+        {{< copyable "shell-regular" >}}
+
+        ``` shell
+        helm install charts/tidb-drainer --name=<release-name> --namespace=<namespace>
+        ```
+
+3. 创建 Backup/Restore 资源对象并开启 TLS。
```
Comment (weekface) [inline zh/enable-tls-between-components.md]: @shuijing198799 PTAL

File: zh/enable-tls-between-components.md [hunk 2]
```diff
@@ -0,0 +1,1068 @@
...
+    ``` shell
+    mkdir -p ~/<cluster-name>-cluster-client-tls
+    cd ~/<cluster-name>-cluster-client-tls
+    kubectl get secret -n <namespace> <cluster-name>-cluster-client-secret  -ojsonpath='{.data.tls\.crt}' | base64 --decode > tls.crt
+    kubectl get secret -n <namespace> <cluster-name>-cluster-client-secret  -ojsonpath='{.data.tls\.key}' | base64 --decode > tls.key
+    kubectl get secret -n <namespace> <cluster-name>-cluster-client-secret  -ojsonpath='{.data.ca\.crt}' | base64 --decode > ca.crt
+    ```
+
+3. 使用 pd-ctl 连接 PD 集群。
+    由于我们刚才在配置 PD Server 端证书的时候，自定义填写了一些 `hosts`，所以需要通过这些 `hosts` 来连接 PD 集群。
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md]:
```suggestion
    
    由于我们刚才在配置 PD Server 端证书的时候，自定义填写了一些 `hosts`，所以需要通过这些 `hosts` 来连接 PD 集群。
```

File: zh/enable-tls-between-components.md [hunk 3]
```diff
@@ -0,0 +1,1068 @@
...
+       config: {}
+     pump:
+       baseImage: pingcap/tidb-binlog
+       replicas: 1
+       requests:
+         storage: "1Gi"
+       config: {}
+    ---
+    apiVersion: pingcap.com/v1alpha1
+    kind: TidbMonitor
```
Comment (DanielZhangQD) [inline zh/enable-tls-between-components.md]: No TLS config for monitor?

**Comment by ran-huang** [comment]
Please take a look at https://github.com/pingcap/docs-tidb-operator/pull/26. The first step for `generating certificate` looks similar to the first step in this PR. 
Do you think we can cross-reference the other document to shorten this doc?

File: zh/enable-tls-between-components.md [hunk 4]
```diff
@@ -0,0 +1,1068 @@
...
+        ```
+
+## 第二步：部署 TiDB 集群
+
+1. 创建一套 TiDB 集群。
+    接下来将会通过两个 CR 对象来创建一个 TiDB 集群，并且执行以下步骤：
+
+    - 为 TiDB 组件间开启 TLS；
+    - 部署一套监控系统；
+    - 部署 Pump 组件。
```
[the remark this answers is not in the document]
Comment (weekface) [reply zh/enable-tls-between-components.md]: 没有遗漏，都包含在 `tidb-cluster.yaml` 文件里了。

**Comment by weekface** [comment]
@ran-huang Good suggestion，不过还是有些细微区别，这个 PR 里的 `usages` 包含 `client auth`，#26 不包含。

File: zh/enable-tls-between-components.md [hunk 5]
```diff
@@ -0,0 +1,1074 @@
...
+## 第二步：部署 TiDB 集群
+
+在这一步中，需要完成以下操作：
+
+    - 创建一套 TiDB 集群
+    - 为 TiDB 组件间开启 TLS
+    - 部署一套监控系统
+    - 部署 Pump 组件
+
+1. 创建一套 TiDB 集群：
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md]: @weekface 可以在这一行加上一句，说明`监控`和`pump`已包含在内。

**Comment by ran-huang** [comment]
LGTM @weekface You can merge this PR if you don't have further updates

**Pull request merged**